qutebrowser/browser/commands.py

```diff
@@ -1177,7 +1177,8 @@ def tab_move(self, index: typing.Union[str, int] = None, count=None):
 
     @cmdutils.register(instance='command-dispatcher', scope='window',
                        maxsplit=0, no_replace_variables=True)
-    def spawn(self, cmdline, userscript=False, verbose=False, detach=False):
+    def spawn(self, cmdline, userscript=False, verbose=False,
+            output_to_tab=False, detach=False):
```

Comment:
The `--output-to-tab` name seems a bit verbose to me - what do you think about simply using `output`?
(Same for other places where `output_to_tab` is used)

Reply:
I went originally with output-to-tab because of the original issue suggesting that, but I definitely agree with the change.

Reply:
Blame @pkillnine :wink: